### ISSUE TITLE
Final Day Tasks

### ISSUE THREAD

**Issue by kpn678** [kpn678/fitlit#57]
Accessibility:
- [x] You must be able to tab through your app and use it without a mouse (can tab, but can't hit enter)
- [x] Your app must still be usable when tested with a colorblind extension
- [x] You must score as close to 100% as possible with the “Lighthouse Accessibility Audit”. Be prepared to explain any accessibility audits your application is failing.
- [x] Your HTML must be written semantically and should use ARIA tags (ONLY if needed / appropriate)

Error Handling:
- [x] Make sure user fills out all forms
- [x] Create function to encompass error messages
- [x] Make sure it is the correct data type?

Misc:
- [x] Ask about date at the top when posting info
- [x] Investigate sleep and activity charts on bigger screen
- [x] Consider moving charts to separate file (not doing)
